components/post-editor/PostEditor.tsx

```diff
@@ -17,6 +17,7 @@ import ImageUploader from '@/components/image-uploader/ImageUploader';
 import ImagePreview, {
   TFireReaderResult,
 } from '@/components/image-preview/ImagePreview';
+import { toast } from 'react-toastify';
```

Comment:
where is it used in this component? Perhaps it got forgotten

Reply:
removed. it is a remain from the first tests.